### PR TITLE
fix(twap): display a warning when fbh was changed after twap order

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Since fb handler might be changed in a random moment, we should handle the case.
We recently added the fb handler verification cache invalidation - #5200.
In this PR I added a banner which is displayed when fb handler is not installed and there are open TWAP orders.

I know, the banner is ugly, @fairlighteth please help 🙏 

image

image

# To Test

1. Create a TWAP order (along with it you should have set thw fallback handler)
2. Reset the fallback handler
3. Update page (might take up to 10 mins due to cache)
- [ ] The banner should be displayed
- [ ] Click to "Update fallback handler" should create a transaction with fbh setup
- [ ] After tx creation the banner should transit to loading state
- [ ] After tx mining the banner should dissapear

If you need to reset a fn handler, just create a transaction via transaction builder:
 - target: your safe address
 - raw data: `0xf08a032300000000000000000000000075cf11467937ce3f2f357ce24ffc3dbf8fd5c226`

And don't forget to remove `fallbackHandlerVerification` localStorage key after that

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]
LGTM in terms of functionality. Waiting for UI changes :)

**Comment by shoom3301** [comment]
Merged #5204 PR from @fairlighteth with design changes in this PR

File: apps/cowswap-frontend/src/modules/twap/containers/SetupFallbackHandlerWarning/index.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+  const updateFallbackHandlerVerification = useSetAtom(updateFallbackHandlerVerificationAtom)
+
+  const extensibleFallbackContext = useExtensibleFallbackContext()
+
+  const handleUpdateClick = async () => {
+    const txHash = await setupFallbackHandler()
+
+    if (txHash) {
+      setPendingTxHash(txHash)
```
Comment (anxolin) [inline apps/cowswap-frontend/src/modules/twap/containers/SetupFallbackHandlerWarning/index.tsx:80]: A re-render of this would loose the posted TX, right?

**Review by anxolin — APPROVED** [review]
Soft approve. Really great addition. Thanks a lot for handling this 🙏

File: apps/cowswap-frontend/src/modules/twap/containers/SetupFallbackHandlerWarning/index.tsx (hunk 1 above)
Comment (shoom3301) [reply apps/cowswap-frontend/src/modules/twap/containers/SetupFallbackHandlerWarning/index.tsx:83]: Nice catch!
Fixed it: https://github.com/cowprotocol/cowswap/pull/5202/commits/35f1802155d78f8992ded18aa6937c44628ebac0
Thanks!

**Comment by elena-zh** [comment]
LGTM

**Pull request merged**